build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         kotlin_stdlib = "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
 
         androidSettings = [
-                min_sdk_version    : 19,
```

Comment:
Product will have to be informed & documentation updated:
> The ReachFive SDK supports Android SDK from API Level 19 (Android version 4.4 KitKat).

Reply:
@jcbee is it acceptable to deprecated android 4.4 KitKat support for our SDK ?

Reply:
Yes it is.